.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: lint
+
+on: push
+
+jobs:
+  lint:
+    name: Lint k8s-charts
```

Comment:
Why do we call it lint? We are also running integration tests on the changed chart.yaml files.

Reply:
What do you mean? Where do you see integration tests in this repo?

Reply:
https://github.com/commercetools/k8s-charts/blob/28da58bbc51eaca21ba65ebe2c9804b987e24e7f/.github/workflows/lint.yml#L13

Reply:
This is not integration test. This installs lint stuff for yaml like yamale.